RTClib.cpp

```diff
@@ -1193,7 +1193,7 @@ DateTime RTC_DS3231::now() {
   uint8_t hh = bcd2bin(Wire._I2C_READ());
   Wire._I2C_READ();
   uint8_t d = bcd2bin(Wire._I2C_READ());
-  uint8_t m = bcd2bin(Wire._I2C_READ());
+  uint8_t m = bcd2bin(Wire._I2C_READ() & 0x7F);
```

Comment:
The datasheet's register address map shows bits 6 and 5 of the month register as always 0. So a more appropriate mask may be `0x1F`.
Reference: `Figure 1` here: https://datasheets.maximintegrated.com/en/ds/DS3231.pdf